siriuspy/siriuspy/devices/idff.py

```diff
@@ -479,7 +479,7 @@ def get_idffclass(devname):
         return None
 
 
-class IDFF(_DeviceSet):
+class IDFF(_DeviceSet, IDFFCtrlBase):
```

Comment:
I noted `IDFFCtrlBase is derived from the `Device` class and here you are also using `DeviceSet`. I'm not sure whether this inheritance is fully compatible. Are you sure this works?